src/components/Accounts/Orders/OrdersList/UserRecentOrder.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
-import moment from 'moment/min/moment-with-locales.min.js';
-import 'moment/locale/en-gb';
+import moment from 'moment';
+// import moment from 'moment/min/moment-with-locales.min.js';
```

Comment:
@IntegralTeam not sure we need those comments here

Reply:
You're right. Removed them.